CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# Contributing to docker-open-banking
```

Comment:
@kaumini has to be Kubernetes rather than Docker. Applies to all cases within this doc.

Reply:
Fixed with 608b8809391c525ac268f8d9da185c4b0d8d539a